# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../inkscape/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- inkscape-0.46/src/dom/io/uristream.cpp.vanilla	2008-07-15 10:37:47.000000000 +0200
+++ inkscape-0.46/src/dom/io/uristream.cpp	2008-07-15 10:37:56.000000000 +0200
@@ -42,7 +42,7 @@
 
 #include "uristream.h"
 
-
+#include <string.h>
 
 namespace org
 {
